parlai/core/torch_agent.py

```diff
@@ -299,7 +299,7 @@ def get_history_str(self):
         Return the string version of the history.
         """
         if self.temp_history and self.history_strings:
-            logging.warning('temporary history strings now include the delimiter.')
+            logging.warn('temporary history strings now include the delimiter.')
```

Comment:
i keep trying to do both.

We should just make a helper method in logging that allows warning

Reply:
done